### PR TITLE
Add D-NeRF dataset download

### PR DESCRIPTION
**Pull request by JulianKnodt**
Adds the ability to download the D-NeRF dataset for dynamic NeRFs.

I know there's currently no way to render dynamic nerf modules, but I would like to attempt adding them later.

### CODE REVIEW

File: scripts/downloads/download_data.py [hunk 1]
```diff
@@ -149,6 +168,10 @@ def main(
         download_nerfstudio(save_dir, capture_name)
     if dataset == "record3d":
         download_record3d(save_dir, "bear")
+    if dataset == "dnerf":
+        if capture_name is not None:
+            console.print("Capture name is ignored when downloading from the dnerf dataset.")
```
Comment (tancik) [inline scripts/downloads/download_data.py:173]: `console` should be capitalized

**Review by tancik — COMMENTED** [review]
LGTM! Thanks for the addition, it would certainly be be nice to get dynamics integrated into this repo.

**Comment by tancik** [comment]
Also take a look at https://docs.nerf.studio/en/latest/reference/contributing.html for instructions on running test and committing code.

**Review by tancik — APPROVED** [review]

**Comment by JulianKnodt** [comment]
Just FYI, I don't have permissions to merge this, so was wondering if you were planning on merging or is this an alternative procedure?

**Comment by tancik** [comment]
> Just FYI, I don't have permissions to merge this, so was wondering if you were planning on merging or is this an alternative procedure?

Hmm, you should have permission since I approved it. Ill merge it for you and look into this later.

**Pull request merged**